src/RecoveryServices/RecoveryServices/help/Set-AzRecoveryServicesVaultProperty.md

```diff
@@ -13,28 +13,45 @@ Updates properties of a Vault.
 
 ## SYNTAX
 
+### AzureRSVaultSoftDelteParameterSet (Default)
 ```
 Set-AzRecoveryServicesVaultProperty -SoftDeleteFeatureState <String> [-VaultId <String>]
  [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
+### AzureRSVaultCMKParameterSet
+```
+Set-AzRecoveryServicesVaultProperty -EncryptionKeyID <string> -KeyVaultSubscriptionId <string> [-InfrastructureEncryption] [-VaultId <String>]
```

Comment:
We talked about specifying the key by vault name + key name + (optional) key version, which should be a third parameter set. But it's OK if you choose to add that in the next sprint

Reply:
We are interested in first trying it this way, we can add it next CMK update. Also we aren't sure of hardcoding the Key url because it might use different format in service. is there a way we can construct the exact key.Id with these three values?

Reply:
It's basically `$"https://{KeyVaultName}.{KeyVaultDnsSuffix}/keys/{key-name}"` with optional `"/{key-version}"` at the end.

You can get `{KeyVaultDnsSuffix}` by `DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.AzureKeyVaultDnsSuffix)`

Reply:
Great we would want to add this parameter set later.